joblib/pool.py

```diff
@@ -1,4 +1,4 @@
-"""Custom implementation of multiprocessing.Pool with custom pickler
+"""Custom implementation of multiprocessing.Pool with custom pickler.
```

Comment:
Just for the record, the period seems required by pep257. https://www.python.org/dev/peps/pep-0257/#one-line-docstrings